pure_utils/strings.go

```diff
@@ -1,9 +1,41 @@
 package pure_utils
 
-import "unicode"
+import (
+	"strings"
+	"unicode"
+
+	fuzzy "github.com/paul-mannino/go-fuzzywuzzy"
+	"golang.org/x/text/runes"
+	"golang.org/x/text/transform"
+	"golang.org/x/text/unicode/norm"
+)
 
 func Capitalize(str string) string {
 	runes := []rune(str)
 	runes[0] = unicode.ToUpper(runes[0])
 	return string(runes)
 }
+
+func Normalize(s string) string {
+	result, _, _ := transform.String(norm.NFC, s)
+	return result
+}
+
+func NormalizeAndRemoveDiacritics(s string) string {
+	t := transform.Chain(
+		norm.NFD,
+		runes.Remove(runes.In(unicode.Mn)),
+		norm.NFC,
+	)
+	result, _, _ := transform.String(t, s)
+	return result
+}
+
+func CleanseString(s string) string {
+	// - normalize
+	// - remove diacritics
+	// - set to lower case
+	// - keep only letters and numbers
+	// - keep non-ASCII characters
+	return strings.TrimSpace(fuzzy.Cleanse(NormalizeAndRemoveDiacritics(s), false))
+}
```

Comment:
IMO, you should put comments on top of the func, so it's displayed on hover by the IDE (where not in Python 😛)
```suggestion
// - normalize
// - remove diacritics
// - set to lower case
// - keep only letters and numbers
// - keep non-ASCII characters
func CleanseString(s string) string {
	return strings.TrimSpace(fuzzy.Cleanse(NormalizeAndRemoveDiacritics(s), false))
}
```

Reply:
aha, yes makes sense

Reply:
same on the class FuzzyMatch